man/mta-sts-daemon.yml.5.adoc

```diff
@@ -47,6 +43,14 @@ The file is in YAML syntax with the following elements:
  ** Options for _redis_ type:
   *** All parameters are passed to `aioredis.create_redis_pool` [0]. Check there for a parameter reference.
 
+*proactive_policy_fetching*::
+
+* *enabled*: (_bool_) enable proactive policy fetching in the background. Default: false
+* *interval*: (_int_) if proactive policy fetching is enabled, it is scheduled every this many seconds.
+It is unaffected by `cache_grace` and vice versa. Default: 86400
+* *concurrency_limit*: (_int_) the maximum number of concurrent domain updates. Default: 100
+* *grace_ratio*: (_int_) proactive fetch for a particular domain is skipped if its cached policy age is less than `interval/grace_ratio`. Default: 2
```

Comment:
Can we use `float` as `grace_ratio` type? I guess current code will accept float values as well and it'll be much more flexible.

Reply:
Sure thing, I'll update it.